### PR TITLE
HAWKULAR-106 availability metric is not collected for non-localhost

### PR DESCRIPTION
**Pull request by ppalaga**
hawkular instances

### CODE REVIEW

**Comment by ppalaga** [comment]
@pilhuhn  could you please review?

File: modules/end-to-end-test/pom.xml [hunk 1]
```diff
@@ -126,6 +124,18 @@
               <version>${project.version}</version>
               <skip>${skipTests}</skip>
               <startupTimeout>240</startupTimeout>
+              <port>${hawkular.management.port}</port>
+              <javaOpts>
+                <javaOpt>-server</javaOpt>
+                <javaOpt>-Xms64m</javaOpt>
+                <javaOpt>-Xmx512m</javaOpt>
+                <javaOpt>-XX:MaxPermSize=256m</javaOpt>
```
Comment (pilhuhn) [inline modules/end-to-end-test/pom.xml]: No longer needed (valid) on Java8

File: modules/pinger/src/main/java/org/hawkular/component/pinger/PingerConfiguration.java [hunk 2]
```diff
@@ -17,32 +17,34 @@
...
+     */
+    public static PingerConfiguration getInstance() {
+        return INSTANCE;
     }
 
-    private final String inventoryBaseUri;
     private final String metricsBaseUri;
 
-    public PingerConfiguration(String inventoryBaseUri, String metricsBaseUri) {
+    public PingerConfiguration(String metricsBaseUri) {
```
Comment (pilhuhn) [inline modules/pinger/src/main/java/org/hawkular/component/pinger/PingerConfiguration.java]: Do you expect client classes to create new instances of the configuration?

File: modules/end-to-end-test/pom.xml (hunk 1 above)
Comment (ppalaga) [inline modules/end-to-end-test/pom.xml]: +1

File: modules/pinger/src/main/java/org/hawkular/component/pinger/PingerConfiguration.java (hunk 2 above)
Comment (ppalaga) [inline modules/pinger/src/main/java/org/hawkular/component/pinger/PingerConfiguration.java]: +1

**Pull request merged**